### PR TITLE
Mount contribution calc 0.0.1

### PR DESCRIPTION
**Pull request by Tr4pSt3R**

### CODE REVIEW

File: app/controllers/contribution_calculator_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class ContributionCalculatorController < EmbeddedToolsController
+  protected
+  
+    def category_id
```
Comment (benlovell) [inline app/controllers/contribution_calculator_controller.rb]: Undent this one level. Also, remove the commented text or return the literal.

File: spec/requests/tool_integration/contribution_calculator_spec.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+RSpec.describe ToolMountPoint::ContributionCalculator, type: :request, features: [:workplace_pension_contribution_calculator] do
```
Comment (benlovell) [inline spec/requests/tool_integration/contribution_calculator_spec.rb]: Unless I'm mistaken, the `features` you've specified doesn't match with the feature flag in your routes.

Comment (Tr4pSt3R) [inline spec/requests/tool_integration/contribution_calculator_spec.rb]: well spotted. Thanks @benlovell. Will get that rectified

**Comment by tomas-stefano** [comment]
Lgtm now. :+1:

**Comment by darokel** [comment]
:+1:

**Pull request merged**